src/main/java/org/example/repository/HibernateUtil.java

```diff
@@ -0,0 +1,20 @@
+package org.example.repository;
+
+import lombok.Getter;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+
+public class HibernateUtil {
+    @Getter
+    private static final SessionFactory sessionFactory;
+
+    static {
+        try {
+            sessionFactory = new Configuration().configure().buildSessionFactory();
+        } catch (Throwable ex) {
```

Comment:
throwable will catch all exceptions. It is a bad practice. In the catch block, you need to declare an exception that is expected